### ISSUE TITLE
Glider adjustment in space 

### ISSUE THREAD

**Issue by Klaranth** [pandorabox-io/in-game#117]
ThinkSome
Gliders shouldn't work on the Moon or at asteroids and their effect should be miniscule on Mars.

i_am_me
Just make them close while inside a vacuum as they should work fine inside a pressurized room on the moon.

Hedgehog
Gliders work in space due to the use of solar winds.
So, they should not work at night, but perfectly usable during the day.

**Comment by S-S-X** [comment]
> So, they should not work at night, but perfectly usable during the day.

Also day / night cycle only applies to earth / moon / mars...

**Comment by OgelGames** [comment]
Should be quite simple to add a periodic check for air/vacuum nodes around glider.

**Comment by SwissalpS** [comment]
note that ``planet_mars:mars_lights`` (or whatever they are called) should be considered as ``air`` for backward compatability with areas that still have them. Maybe also ``technic:dummy_lights``.
beacon beams are climbable, so probably not going to be an issue.

**Comment by Klaranth** [comment]
skylord/skypro 
The "handglider" mod acts more of an parachute.. I dont really think its as realistic as it would be.. That's why I think renaming the "handglider" item name from "Glider" to "Parachute" and maybe you could replace the glider with this: https://github.com/skylord-at-hub/glider/edit/master/Readme.md
This is my repo.. I have done a few bug fixes so then the mod doesn't crash the server :)

SX 
Check for player, reduce visuals and made boosters OP? Don't think that's worth it to replace stuff but might be for updating mod.
Bug fixes you should probably open PR for each. Visuals you should probably make configurable and then PR, same for booster.
To make it not work as a parachute you could make it sink faster depending on lateral speed.

The4spaceconstants 
handglider makes you go faster in air

et 
there was that one mod in nonsensical skyblock that had an insanely good glider
fast but dangerous, as it should be

**Comment by SwissalpS** [comment]
> The "handglider" mod acts more of an parachute.. I dont really think its as realistic as it would be.. That's why I think renaming the "handglider" item name from "Glider" to "Parachute"

I disagree with this pretext. Parachutes don't provide much steering capabilities. The gliders on pandorabox can be steered very well. IRL gliders tend to loose height constantly unless the pilot finds upwinds.
